xarray_sentinel/esa_safe.py

```diff
@@ -1,6 +1,7 @@
 import functools
 import os
 import pathlib
+import re
```

Comment:
flake8 returns: `F401 're' imported but unused`

flake8 errors are now failures in the CI, maybe we need to make them so.

Reply:
Thanks I fux it